lib/python/rose/bush.py

```diff
@@ -57,25 +58,33 @@ class RoseBushService(object):
     def __init__(self, *args, **kwargs):
         self.exposed = True
         self.suite_engine_proc = SuiteEngineProcessor.get_processor()
-        self.host_name = HostSelector().get_local_host()
-        if self.host_name and "." in self.host_name:
-            self.host_name = self.host_name.split(".", 1)[0]
+        rose_conf = ResourceLocator.default().get_conf()
+        self.title = rose_conf.get_value(["rose-bush", "title"], self.TITLE)
+        self.host_name = rose_conf.get_value(["rose-bush", "host"])
+        if self.host_name is None:
+            self.host_name = HostSelector().get_local_host()
+            if self.host_name and "." in self.host_name:
+                self.host_name = self.host_name.split(".", 1)[0]
         self.rose_version = ResourceLocator.default().get_version()
-        res_loc = ResourceLocator.default()
         template_env = jinja2.Environment(loader=jinja2.FileSystemLoader(
             ResourceLocator.default().get_util_home(
                 "lib", "html", "template", "rose-bush")))
         self.template_env = template_env
 
     @cherrypy.expose
-    def index(self):
+    def index(self, form=None):
         """Display a page to input user ID and suite ID."""
         # TODO: some way to allow autocomplete of user field?
+        data = {
+            "title": self.title,
+            "host": self.host_name,
+            "rose_version": self.rose_version,
+            "script": cherrypy.request.script_name,
+        }
+        if form == "json":
```

Comment:
`format` is a python built-in name. We didn't think about this when we were doing `rosie.ws`. When we wrote Rose Bush, we have chosen `form=` for all methods to avoid clashing with the built-in.